pyk/src/tests/unit/kast/test_outer_parser.py

```diff
@@ -382,15 +384,42 @@ def test_import(k_text: str, expected: AST) -> None:
 
 
 MODULE_TEST_DATA: Final = (
-    ('module FOO endmodule', Module('FOO')),
-    ('module FOO [foo] endmodule', Module('FOO', att=Att((('foo', ''),)))),
-    ('module FOO imports BAR endmodule', Module('FOO', imports=(Import('BAR'),))),
-    ('module FOO imports BAR imports BAZ endmodule', Module('FOO', imports=(Import('BAR'), Import('BAZ')))),
-    ('module FOO rule x endmodule', Module('FOO', sentences=(Rule('x'),))),
-    ('module FOO rule x rule y endmodule', Module('FOO', sentences=(Rule('x'), Rule('y')))),
+    ('module FOO endmodule', Module('FOO', att=Att(((LOCATION, '1,1,1,21'),)))),
+    (
+        'module FOO [foo] endmodule',
+        Module(
+            'FOO',
+            att=Att(
+                (
+                    ('foo', ''),
+                    (LOCATION, '1,1,1,27'),
+                )
+            ),
+        ),
+    ),
+    ('module FOO imports BAR endmodule', Module('FOO', imports=(Import('BAR'),), att=Att(((LOCATION, '1,1,1,33'),)))),
+    (
+        'module FOO imports BAR imports BAZ endmodule',
+        Module('FOO', imports=(Import('BAR'), Import('BAZ')), att=Att(((LOCATION, '1,1,1,45'),))),
+    ),
+    ('module FOO rule x endmodule', Module('FOO', sentences=(Rule('x'),), att=Att(((LOCATION, '1,1,1,28'),)))),
+    (
+        'module FOO rule x rule y endmodule',
+        Module('FOO', sentences=(Rule('x'), Rule('y')), att=Att(((LOCATION, '1,1,1,35'),))),
+    ),
     (
         'module FOO [foo] imports BAR rule x endmodule',
-        Module('FOO', sentences=(Rule('x'),), imports=(Import('BAR'),), att=Att((('foo', ''),))),
+        Module(
+            'FOO',
+            sentences=(Rule('x'),),
+            imports=(Import('BAR'),),
+            att=Att(
+                (
+                    ('foo', ''),
+                    (LOCATION, '1,1,1,46'),
+                )
+            ),
+        ),
```

Comment:
This all looks good; could we perhaps have a unit test with multiple lines using a docstring?

Reply:
Great!